### PR TITLE
fix: defer exit to collect init containers' logs

### PR DESCRIPTION
**Pull request by zhujian7**
**What this PR does / why we need it**:

defer exit to collect init containers' logs

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #

Reference to #

**Special notes for your reviewer**:

/cc @cd1989 @supereagle 

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: build/toolbox/Dockerfile [hunk 1]
```diff
@@ -4,4 +4,7 @@ LABEL maintainer="zhujian@caicloud.io"
 
 WORKDIR /usr/bin/cyclone-toolbox
 
+RUN apk update && \
+    apk add coreutils
```
Comment (cd1989) [inline build/toolbox/Dockerfile]: `–no-cache`

Comment (zhujian7) [reply build/toolbox/Dockerfile]: have fixed, PTAL

**Pull request merged**